src/consoleStream.ts

```diff
@@ -12,13 +11,13 @@ const createConsoleWriteStream = (options: LogflareUserOptionsI) => {
 
   writeStream._write = (chunk: any, encoding: any, callback: any) => {
     const batch = Array.isArray(chunk) ? chunk : [chunk]
-    _(batch)
-      .map(JSON.parse)
+    batch
+      .map((chunkItem) => JSON.parse(chunkItem))
```

Comment:
I could not pass `JSON.parse` alone here, as was being done before, because second parameter to JSON.parse is a "reviver" function, whereas the second parameter of the map function is a number (the index of the current item), so the types misaligned.